src/lib/pages/frontsite/PatronProfile/PatronCurrentDocumentRequests/PatronCurrentDocumentRequests.js

```diff
@@ -233,9 +228,37 @@ class PatronCurrentDocumentRequests extends Component {
     this.setState({ isSuccessMessageVisible: false, successMessage: '' });
   };
 
+  onShowAll = () => {
+    const { patronPid, fetchPatronDocumentRequests } = this.props;
+    fetchPatronDocumentRequests(patronPid, {
```

Comment:
why do we need to pass patronPid here, but not below?

Reply:
Is not the same function call. This first one calls the function that was passed as prop to the component.
The second calls a function of the component that also calls the passed one with the appropriate parameters.